spec/mongoid/scopable_spec.rb

```diff
@@ -548,6 +566,58 @@ class << Band
               expect(criteria.selector).to eq({ "origin" => "England" })
             end
           end
+
+          context "when chaining scopes through more than one model" do
+
+            before do
+              Author.scope(:author, -> { where(author: true) } )
+              Article.scope(:is_public, -> { where(public: true) } )
+              Article.scope(:authored, -> {
+                author_ids = Author.author.pluck(:id)
+                where(:author_id.in => author_ids)
+              })
+
+              Author.create(author: true, id: 1)
+              Author.create(author: true, id: 2)
+              Author.create(author: true, id: 3)
+              Article.create(author_id: 1, public: true)
+              Article.create(author_id: 2, public: true)
+              Article.create(author_id: 3, public: false)
+            end
+
+            after do
+              class << Article
+                undef_method :is_public
+                undef_method :authored
+              end
+              Article._declared_scopes.clear
+              class << Author
+                undef_method :author
+              end
+              Author._declared_scopes.clear
+            end
+
+            context "when calling another model's scope from within a scope" do
+
+              let(:authored_count) do
+                Article.authored.size
+              end
+
+              it "correct documents returned" do
```

Comment:
It'd be great if this read like a sentence:

it 'returns the correct documents' do